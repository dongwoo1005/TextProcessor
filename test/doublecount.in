stdin count a count b
abbababbababb
