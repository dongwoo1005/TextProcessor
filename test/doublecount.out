1122334546578
